packages/fuels-signers/src/lib.rs

```diff
@@ -147,30 +147,46 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn send_transfer_transactions() -> fuels_types::errors::Result<()> {
+    async fn test_wallet_get_coins() -> fuels_types::errors::Result<()> {
+        const AMOUNT: u64 = 1000;
+        const NUM_COINS: u64 = 3;
+        let mut wallet = WalletUnlocked::new_random(None);
+        let coins = setup_single_asset_coins(wallet.address(), BASE_ASSET_ID, NUM_COINS, AMOUNT);
+
+        let (client, _) = setup_test_client(coins, vec![], None, None, None).await;
+        let provider = Provider::new(client);
+        wallet.set_provider(provider.clone());
+
+        let wallet_initial_coins = wallet.get_coins(BASE_ASSET_ID).await?;
+        let total_amount: u64 = wallet_initial_coins.iter().map(|c| c.amount).sum();
+
+        assert_eq!(wallet_initial_coins.len(), NUM_COINS as usize);
+        assert_eq!(total_amount, AMOUNT * NUM_COINS);
+
+        Ok(())
+    }
+
+    async fn setup_transfer_test(amount: u64) -> (WalletUnlocked, Wallet) {
         // Setup two sets of coins, one for each wallet, each containing 1 coin with 1 amount.
         let mut wallet_1 = WalletUnlocked::new_random(None);
         let mut wallet_2 = WalletUnlocked::new_random(None).lock();
 
-        let amount = 1000000;
-        let mut coins_1 = setup_single_asset_coins(wallet_1.address(), BASE_ASSET_ID, 1, amount);
-        let coins_2 = setup_single_asset_coins(wallet_2.address(), BASE_ASSET_ID, 1, amount);
-
-        coins_1.extend(coins_2);
+        let coins = setup_single_asset_coins(wallet_1.address(), BASE_ASSET_ID, 1, amount);
```

Comment:
Now, that we do not add coins to the second wallet the comments on lines `170` and `176` are not valid anymore.